src/kernelTypes/convCircKernel.jl

```diff
@@ -0,0 +1,92 @@
+export convCircKernel ,getConvCircKernel
+## For the functions nImgIn, nImgOut, nFeatIn, nFeatOut, nTheta, getOp, initTheta : see abstractConvKernel.jl
+## All convKernel types are assumed to have fields nImage and sK
+mutable struct convCircKernel{T} <: abstractConvKernel{T}
+    nImg :: Array{Int,1}
+    sK   :: Array{Int,1}
+    iK   :: Array{Int,1}
+end
+
+function nImgOut(this::convCircKernel)
+    return [this.nImg[1]; this.nImg[2]; this.sK[3]]
+ end
+
+function getConvCircKernel(TYPE::Type,nImg,sK)
+ 
+    n3 = [nImg[1],nImg[2],sK[3]]
+    Ind = zeros(n3[1],n3[2],n3[3])
```

Comment:
If `n3` ever has more elements and you're tired of writing this out, `zeros(n3...)` does the same thing. It's called 'splatting'

Reply:
cool. good to know. it that the same as `tuple(n3...)` ?

Reply:
Yup. It only works for function calls, but it just passes each element of `n3` as a different argument.

The opposite can be done in function definitions, and is called 'slurping'. You can collect any number of input arguments into a tuple. For example below I'll slurp up the variable input then splat it into the addition operator.

```
julia> function foo(x...)
           println(x)
           +(x...)
       end
foo (generic function with 1 method)

julia> foo(1,2,3)
(1, 2, 3)
6

julia> foo(1,2,3,4,5,6)
(1, 2, 3, 4, 5, 6)
21

julia> foo(1,2,3,4,5,6,3,4,5,6,5,3,34,4)
(1, 2, 3, 4, 5, 6, 3, 4, 5, 6, 5, 3, 34, 4)
85
```